stingray/events.py

```diff
@@ -108,13 +108,18 @@ def to_lc(self, dt, tstart=None, tseg=None):
 
         tseg: float
             Total duration of light curve
-        
+
         Returns
         -------
         lc: `Lightcurve` object
         """
 
-        return Lightcurve.make_lightcurve(self.time, dt, tstart=tstart, tseg=tseg)
+        if tstart is None and self.gti is not None:
+            tstart = self.gti[0][0]
+            tseg = self.gti[-1][1] - tstart
```

Comment:
So this always uses gtis for making a light curve? I'm assuming that's intended?